[IMP] helpdesk: update flow and images for reinvoicing time

mivu-odoo opened the pull request:
Rewrote steps because this was an outdated flow for Odoo v14 and updated images to match new steps.
Old steps: Configuration --> Create a sales order and invoice
New steps: Config. --> create SO --> record timesheet --> Link ticket to SO --> create invoice

I was made aware of this outdated flow and the confusion it was causing users thanks to PR #868.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that you can later pull from to bill your client. This gives you more control over what you charge your client, and it is more efficient.
 
 Configuration
 =============
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and select the Helpdesk team you want to track timesheets for. Click **Edit** and enable the options Timesheet on Ticket and Time Reinvoicing.
```
Need a line break after the heading.
```suggestion
Configuration
=============

Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` ...
```

StraubCreative replied:
Italicize Odoo options to indicate that these are choices and features of the software.

```suggestion
Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Under Timesheet on Ticket, choose the Project to which tickets (and timesheets) will be linked by default. Open the Project's external link to edit its settings and enable the Billable option. Save both the Project and the Helpdesk team's settings to apply these features.
```
Contextualize this. "Under Timesheet on Ticket..." could mean many things. Under _what_ exactly?

```suggestion
In the Timesheet on Ticket options
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
 
-Create a sales order and an invoice
-====================================
+Create a sales order
+====================
+After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Add the customer from the ticket to the new quotation. In the Order Lines tab, add the Service on Timesheet product. Click **Confirm** to turn the quotation into a sales order. Take note of the :abbr:`SO (Sales Order)` name for later.
```
Need a line break after heading.

```suggestion
Create a sales order
====================

After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Add the customer from the ticket to the new quotation. In the Order Lines tab, add the Service on Timesheet product. Click **Confirm** to turn the quotation into a sales order. Take note of the :abbr:`SO (Sales Order)` name for later.
```

StraubCreative replied:
Suggestion to improve linear flow of entire paragraph:

```suggestion
Create a sales order
====================

After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create` and add the customer from the ticket to the new quotation. In the Order Lines tab, add the Service on Timesheet product and then click **Confirm** to turn the quotation into a sales order. Be sure to take note of the :abbr:`SO (Sales Order)` name for later reference.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
+   :align: center
+   :alt: Add Service on Timesheet product to the SO
+
+Record a timesheet
+==================
```
Line break after heading

```suggestion
Record a timesheet
==================

```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
+
+Record a timesheet
+==================
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking **Tickets** on the team's card. You can also access the ticket by going to the Projects dashboard and clicking **Tickets** on the project's card. Click **Edit** and use the Timesheets tab to record the time you spend on the ticket.
```
Capitalize Odoo product (project vs Project).

```suggestion
Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking **Tickets** on the team's card. You can also access the ticket by going to the Projects dashboard and clicking **Tickets** on the Project's card. Click **Edit** and use the Timesheets tab to record the time you spend on the ticket.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
+   :alt: Record time spent on a ticket
 
-*Create Sales Order* and proceed to create the invoice.
+Link the Helpdesk ticket to the SO
+==================================
```
Add line break after heading

```suggestion
Link the Helpdesk ticket to the SO
==================================

```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
-*Create Sales Order* and proceed to create the invoice.
+Link the Helpdesk ticket to the SO
+==================================
+Still on the ticket form, click **Edit** and click the Sales Order Item field. Select the :abbr:`SO (Sales Order)` that you created earlier from the dropdown menu. Click **Save**.
```
Small suggestion to improve flow/readability. 

```suggestion
While still on the ticket form, click **Edit** and then click the Sales Order Item field. Select the :abbr:`SO (Sales Order)` that you created earlier from the dropdown menu. Click **Save**.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
-   :alt: Sales Order from a task in Odoo Helpdesk
+   :alt: Link the SO item to the ticket
+
+Create the invoice
+==================
```
Add line break after heading

```suggestion
Create the invoice
==================

```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,46 @@
+
+Create the invoice
+==================
+When you solve the Helpdesk ticket and are ready to bill the client for your time, click the **Sales Order** smart button on the ticket form to take you to the :abbr:`SO (Sales Order)`. The Delivered column should match the number of hours recorded on the ticket's timesheet. Click **Create Invoice** to bill your client for the time spent on the ticket.
```
Removing 2nd-person familiar tone in favor of 3rd-person instructional.

```suggestion
When the Helpdesk ticket is completed and the client is ready to be billed for time, begin by clicking the **Sales Order** smart button on the ticket form to navigate to the :abbr:`SO (Sales Order)`. The Delivered column should match the number of hours recorded on the ticket's timesheet. Click **Create Invoice** to bill the client for the time spent on the ticket.
```

StraubCreative reviewed (changes requested):
Hi @mivu-odoo 👋 

Great additions to this doc!

I offered a number of suggestions having to do mostly around tone (2nd vs. 3rd person) and sentence flow, along with a few small requests to fix RST doc structure. Please find them all below.

As well, be sure to [start a new line before the 100th character](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character) per RST guidelines. This applies to the _entire_ document (e.g., specifically lines 5, 9, 11, 19, etc.).

Hope this is helpful and looking forward to your changes!

mivu-odoo commented:
Hi @StraubCreative ! 

I fixed the file so that the headings now have a line break after them. I also added line breaks so I didn't go past the 100th character as per RST guidelines, specified the context for the Timesheet on Ticket drop-down option, and fixed the See Also section to use the doc directive instead of a hyperlink reference. Please review these changes when convenient and let me know if there's anything else I missed. 

Thank you so much again for guiding me throughout this process! I'm definitely learning a lot thanks to you :-)

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that can be
+later pulled from to bill your client. This integration provides more control and transparency over
```
Wording suggestion.
This is the first line of the doc and I had to reread it a couple times to make sense of it. Technically it's correct but it's clunky...can clean up a little.
```suggestion
Connect your Helpdesk ticket to a Project task, and use either one to record timesheets that can
later be pulled from in order to bill the client. This integration provides more control and
transparency over
```

mivu-odoo replied:
I changed the wording and structure a little to be more like chronological steps. Hopefully, that reads smoother!

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
```
Change from `**bold**` format to `:guilabel:`

```suggestion
Click :guilabel:`Edit` and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
 
-.. image:: media/reinvoice_time1.png
+In the *Timesheet on Ticket* drop-down option, choose the Project to which tickets (and timesheets)
```
We _might_ need to change these from `*italics*` to `:guilabel:` since we're talking about UI options here. Can leave for now but let's bring up with Doc Review after.

mivu-odoo replied:
I ended up changing these to a `:guilabel:`, we'll see what Doc Review says later.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
-.. image:: media/reinvoice_time1.png
+In the *Timesheet on Ticket* drop-down option, choose the Project to which tickets (and timesheets)
+will be linked by default. Open the Project's external link to edit its settings and enable the
+*Billable* option. Save both the Project and the Helpdesk team's settings to apply these features.
```
Same here. 
Should `*Billable*` be `:guilabel:` since it's a clickable option on the UI?

mivu-odoo replied:
Also changed to `:guilabel:` here.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+
+After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Add the customer
+from the ticket to the new quotation. In the *Order Lines* tab, add the *Service on Timesheet*
+product. Click **Confirm** to turn the quotation into a sales order. 
```
Change to `:guilabel:`
```suggestion
product. Click :guilabel:`Confirm` to turn the quotation into a sales order. 
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+product. Click **Confirm** to turn the quotation into a sales order. 
+
+.. note::
+   Remember the :abbr:`SO(Sales Order)` name for later.
```
Might be better formatted as a tip actually.
Contextualize further...why remember it? For what purpose?

mivu-odoo replied:
I changed it to be a tip and added that it should be remembered to help the user link their ticket to the correct SO.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
 
-Now, once you have recorded the time you spent on the helpdesk ticket, under the *Timesheets* tab,
-access the task clicking on its name.
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking **Tickets** on the
```
Change to `:guilabel:`

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
-access the task clicking on its name.
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking **Tickets** on the
+team's card. You can also access the ticket by going to the Projects dashboard and clicking
+**Tickets** on the Project's card. Click **Edit** and use the *Timesheets* tab to record the time
```
Change `**Tickets**` and `**Edit**` to `:guilabel:` format.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+==================================
 
-*Create Sales Order* and proceed to create the invoice.
+While still on the ticket form, click **Edit** and then click the *Sales Order Item* field. Select
```
Change `**Edit**` to `:guilabel:`
Might need to change `*Sales Order Item*` too since it's a field...

mivu-odoo replied:
I changed both `Edit` and `Sales Order Item` to have a `:guilabel:`.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
 
-*Create Sales Order* and proceed to create the invoice.
+While still on the ticket form, click **Edit** and then click the *Sales Order Item* field. Select
+the :abbr:`SO (Sales Order)` that you created earlier from the drop-down menu. Click **Save**.
```
`**Save**` --> `:guilabel:`

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+==================
+
+When the Helpdesk ticket is completed and the client is ready to be billed for time, begin by
+clicking the **Sales Order** smart button on the ticket form to navigate to the sales order. The
```
`:guilabel:`

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,67 @@
+When the Helpdesk ticket is completed and the client is ready to be billed for time, begin by
+clicking the **Sales Order** smart button on the ticket form to navigate to the sales order. The
+Delivered column should match the number of hours recorded on the ticket's timesheet. Click
+**Create Invoice** to bill the client for the time spent on the ticket.
```
`:guilabel:`

StraubCreative reviewed (changes requested):
@mivu-odoo thanks for the latest batch of revisions. The first round change requests look like they were addressed so all good there 👍 

For next round, the big change we need to make is to take the standard `**bold**` or `*italic*` formatting we've specifically been using on UI objects (buttons, fields, etc.) and replace those with `:guilabel:` tag. 

I found a bunch of instances where a change is needed and labeled them accordingly in the review, which you can use as a guide, however I'd recommend actually combing through the RST yourself, line-by-line, and looking for every instance where textured formatting is used. If the formatting is applied on a UI feature, then we'd need to change it to `:guilabel:`.

You'll see too I found a few instances that were questionable to me. Feel free to make a call there bc either way will have to address with Doc Review.

Other than that, a couple further word/grammar suggestions.

Thanks again and happy hunting!

mivu-odoo commented:
@StraubCreative Hi Zac, I finished making changes as requested:

- Reworded the introduction so it would be a little less clunky. 
- Changed the `.. note:` to a `.. tip:` and wrote why the user should remember the SO name. 
- Went through and changed UI objects to have a `:guilabel:` instead of bold or italic formatting.
- Noticed the `:menuselection:` sentence in the "Configuration" section got accidentally deleted, so I added that back in.

Let me know how it looks when you get the chance! Thank you :-)

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,74 @@
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect a Helpdesk ticket to a Project task. While working on the ticket, use either the ticket or 
+the task to record timesheets. When the ticket is solved, Odoo can pull from those timesheets to 
+bill the client. This integration provides more control and transparency over what the client is 
+charged for.
```
Still think there's room for improvement here.

Ideally this would flow with a natural rhythm and _succinctly_ say what the doc/feature is about.

However since this integration is a bit nuanced (this part in particular), I don't think short sentence fragments stacked together will work here; short, brief sentences only work if there's a specific direction being given.

To that end, I suggest operationalizing the feature with longer sentences that contain commas and time order words in order to capture the breadth of information we're trying to share while also expressing it in a linear way:

```suggestion
The **Timesheet on Ticket** integration provides more control and transparency over how clients are
charged, and what they're specifically billed for.

The way it works is on the backend of Odoo, Project managers can connect Helpdesk tickets to
Project tasks. While working on the ticket, either the ticket itself or the task can be used to
record timesheets. When the ticket is solved, Odoo can pull from either the task or ticket
timesheets in order to reference billable hours when it's time to bill the client.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,74 @@
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`. Select a Helpdesk team and 
+click :guilabel:`Edit`. Then, enable the options :guilabel:`Timesheet on Ticket` and 
+:guilabel:`Time Reinvoicing`.
```
Small suggestion to improve readability. Might just be style difference, however to me this small change helps connect the flow between different actions in the same breath, as opposed to brief instructions that may or may not appear related.

```suggestion
Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, select a Helpdesk team, and
click :guilabel:`Edit`. Then, enable the options :guilabel:`Timesheet on Ticket` and
:guilabel:`Time Reinvoicing`.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,74 @@
-Now, once you have recorded the time you spent on the helpdesk ticket, under the *Timesheets* tab,
-access the task clicking on its name.
+Jump back to the Helpdesk ticket by going to the Helpdesk dashboard and clicking 
+:guilabel:`Tickets` on the team's card. Another way to access the ticket is by going to the 
+Projects dashboard and clicking :guilabel:`Tickets` on the project card. Click :guilabel:`Edit` 
```
Think we can do something with this sentence that starts with "Another way..."

I dont know if it warrants a `:tip:` directive because that would break up the flow of the instruction.

Maybe something like this?

(Also, I would consider using more time order flow words to help communicate linear and conditional actions...)

```suggestion
:guilabel:`Tickets` on the team's card (or by going to the 
Projects dashboard and clicking :guilabel:`Tickets` on the project card). Then click :guilabel:`Edit` 
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,74 @@
+:guilabel:`Time Reinvoicing`.
 
-.. image:: media/reinvoice_time1.png
+In the :guilabel:`Timesheet on Ticket` drop-down option, choose the Project to which tickets (and 
+timesheets) will be linked by default. Open the Project's external link to edit its settings and 
+enable the :guilabel:`Billable` option. Save both the Project and the Helpdesk team's settings to 
+apply these features.
```
This is only one section I'm highlighting here, but the whole document needs to be cleaned of trailing whitespaces, line by line.

The line break right at the end of a sentence _is_ the whitespace when it's rendered into HTML.

```suggestion
In the :guilabel:`Timesheet on Ticket` drop-down option, choose the Project to which tickets (and
timesheets) will be linked by default. Open the Project's external link to edit its settings and
enable the :guilabel:`Billable` option. Save both the Project and the Helpdesk team's settings to
apply these features.
```

StraubCreative reviewed (changes requested):
@mivu-odoo hey there. I think this pass was really good. Thanks for addressing all of the CR's in the previous review.

I have just a few comments on this pass, mostly high-level items which include:
• revisiting that intro section 
• a couple suggestions on tone/style in a couple spots. 
• a doc formatting issue which I think would get brought up in DR anyway

Thanks and looking forward to the next round of revisions. Think we're almost done!

mivu-odoo reviewed (commented):
@StraubCreative 

Thanks for the feedback. In this change, I did the following:

- Deleted trailing white spaces at the end of the lines
- Cleaned up the introduction using your suggestion
- Added more time order flow words to the overall doc so actions are easier to follow
- Combined first and second sentences in Paragraph 1 of Configuration to show that the steps are connected

Let me know if there's anything else I can still improve on. I'm 50/50 on the introduction right now, my main worry is that it's too long, but I felt like it was useful to explain the real-life flow and emphasize the benefit of the Timesheet on Ticket feature.

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,82 @@
+The :guilabel:`Timesheet on Ticket` integration provides more control and transparency over how
+clients are charged, and what they're specifically billed for.
+
+In Odoo, Project managers can connect Helpdesk tickets to Project tasks. As the Helpdesk agent
+works on the ticket, they can use the ticket itself or the Project task to record timesheets.
+Thanks to the :guilabel:`Timesheet on Ticket` integration, timesheets will always be consistent and
+up-to-date on both records. Once a ticket is solved, the client can be billed for the time spent on
+the ticket. Odoo will pull from either the task or the ticket timesheet to accurately bill the
+client. Keep timesheets organized with the :guilabel:`Timesheet on Ticket` feature. 
```
I see what you're saying about thinking it's too long.
Two suggestions here: one is to add another line break at line 10 to reduce the "wall of text" vibe and then also delete the trailing sentence on line 13 (don't think it's needed?)
```suggestion
In Odoo, Project managers can connect Helpdesk tickets to Project tasks. As the Helpdesk agent
works on the ticket, they can use the ticket itself or the Project task to record timesheets.

Thanks to the :guilabel:`Timesheet on Ticket` integration, timesheets will always be consistent and
up-to-date on both records. Once a ticket is solved, the client can be billed for the time spent on
the ticket. Odoo will pull from either the task or the ticket timesheet to accurately bill the
client.
```

StraubCreative commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,82 @@
+Link the Helpdesk ticket to the SO
+==================================
+
+To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form. Then, click
+:guilabel:`Edit` and click the :guilabel:`Sales Order Item` field. Next, select the :abbr:`SO
+(Sales Order)` that was created earlier from the drop-down menu. Lastly, click :guilabel:`Save` to
+connect the ticket and the :abbr:`SO (Sales Order)`.
```
Suggestion.
First sentence may have been ended too early before an explicit/actionable instruction was given.
Helps the flow/rhythm of the paragraph too imo...

Up to you though as I'm being nitpicky at this point 😉 

```suggestion
To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form, click
:guilabel:`Edit` and then click the :guilabel:`Sales Order Item` field. Next, find and select the :abbr:`SO
(Sales Order)` that was created earlier from the drop-down menu. Lastly, click :guilabel:`Save` to
connect the ticket and the :abbr:`SO (Sales Order)`.
```

StraubCreative reviewed (changes requested):
@mivu-odoo nice cleanup this round 👍 

Almost done I think. Just two more things, that, in all honesty, I would consider optional.

Going to mark as Change Requests that way they're at least addressed but how you want to play it from here is up to you 🤔

mivu-odoo reviewed (commented):
Hi @StraubCreative :-)

Thanks for the feedback! I'm always down for nit-picking haha 

I made the following changes:

- Broke up the intro paragraph w/ a line break before "Thanks to..."
- Deleted the awkward sentence at the end of the intro "Keep timesheets organized..."
- Link the Helpdesk ticket to the SO > 1st paragraph: combined first and second sentences for better flow

Hopefully, the light is at the end of the tunnel and we can close this PR soon :-)

StraubCreative reviewed (approved):
LGTM 🚀

StraubCreative commented:
@mivu-odoo
Before tagging in DR-- 
Need to fix the merge conflict [here](https://github.com/odoo/documentation/pull/1901/conflicts) first.
Same issue as what we saw in [#1982](https://github.com/odoo/documentation/pull/1982#issuecomment-1136412398)...

StraubCreative commented:
No more merge conflict as of [0f0eca5](https://github.com/odoo/documentation/commit/0f0eca5b2d0bcf8e47b6a6e6636d35ddbc93befd).

On to technical review...

AntoineVDV commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,83 @@
+and enable the :guilabel:`Billable` option. Finally, save both the Project and the Helpdesk team's
+settings to apply these features.
+
+.. image:: reinvoice_from_project/project_billable.png
```
I saw that @StraubCreative asked you to swap underscores for hyphens in #1982. If you want to do the same here, take the opportunity to change the commit tag ;)

AntoineVDV reviewed (approved):
Same as for https://github.com/odoo/documentation/pull/1982 for the commit tag.

AntoineVDV reviewed (approved):
@robodoo r+

aju-odoo reviewed (commented):
Hello,

Thanks for your work. I am glad to see this doc being updated, especially since it was often a source of customer tickets.

I left a few comments. Those are only suggestions, do as you please with them.

One important thing to note though: tickets are no longer linked to project tasks. They both live their lives independently from each other.

I hope this helps. Have a nice day :)

mivu-odoo commented on content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst:
```diff
@@ -2,35 +2,82 @@
+The :guilabel:`Timesheet on Ticket` integration provides more control and transparency over how
+clients are charged, and what they're specifically billed for.
+
+In Odoo, Project managers can connect Helpdesk tickets to Project tasks. As the Helpdesk agent
+works on the ticket, they can use the ticket itself or the Project task to record timesheets.
```
Thank you for letting me know! I have opened PR #2204 to apply this change, as well as the other comments you brought up.
